Jenkinsfile

```diff
@@ -449,8 +449,8 @@ pipeline {
       }
       steps {
         script {
-          parallel(cambpmGetMatrixStages('engine-rest', failedStageTypes, { stageType, dbLabel ->
-            return cambpmWithLabels('rest-api')
+          parallel(cambpmGetMatrixStages('engine-rest', failedStageTypes, { allowedStageLabels, dbLabel ->
+            return cambpmWithLabelsList(allowedStageLabels)
```

Comment:
This creates now to additional reads than before, but I guess now the code is consistent.

Reply:
Actually, this reduces the number of reads. The `allowedStageLabels` is obtained on the first read of the `stage-types.yaml` file inside the `cambpmGetMatrixStages` method, and it is stored as a variable. The purpose of the refactoring above was to avoid additional reads of this file.